gnomad/assessment/summary_stats.py

```diff
@@ -397,13 +406,33 @@ def get_summary_stats_variant_filter_expr(
     if filter_expr is not None:
         log_list.append("variants that pass all variant QC filters")
         ss_filter_expr["variant_qc_pass"] = hl.len(filter_expr) == 0
-    if max_af is not None:
-        if isinstance(max_af, float):
-            max_af = [max_af]
-        log_list.extend([f"variants with (AF < {af:.2e})" for af in max_af])
-        ss_filter_expr["max_af"] = hl.struct(
-            **{f"{af}": freq_expr < af for af in max_af}
-        )
+
+    def get_max_filter(
+        max_expr: hl.expr.Float64Expression,
+        max_cutoff: Union[float, List[float], None],
+        name: str = "",
```

Comment:
A bit of a nitpick since this is "private"  but default to "af" here since it was previously just used for max_af and "" doesnt necessary work with the loggers/dictionary key.